integration_tests/mockApis/auth.ts

```diff
@@ -119,6 +119,7 @@ const token = (userToken: UserToken) =>
       },
       jsonBody: {
         access_token: createToken(userToken),
+        auth_source: 'nomis',
```

Comment:
The new middleware checks for the auth source is `nomis` before trying to retrieve case load data (since it's a prison/NOMIS specific thing), so this is important for the integration tests to pass.